SourcetrailExtensionTests/Helpers/TestPathResolver.cs

```diff
@@ -28,9 +28,17 @@ public override string GetCompilationDatabaseFilePath()
 
 		public override string GetAsAbsoluteCanonicalPath(string path, IVCProjectWrapper project)
 		{
-			if (path.Length > 0 && !path.StartsWith("<") && !System.IO.Path.IsPathRooted(path))
+			if (path.Length > 0 && !path.StartsWith("<"))
 			{
-				path = "<ProjectBaseDirectory>/" + path;
+				if (!System.IO.Path.IsPathRooted(path))
+				{
+					path = "<ProjectBaseDirectory>/" + path;
+				}
+				else if (path.StartsWith(project.GetProjectDirectory(), System.StringComparison.CurrentCultureIgnoreCase))
```

Comment:
To work around paths Visual Studio resolved for us, have the tests replace the real project dir with `<Macro $(ProjectDir)>`.